db/models/reviews.js

```diff
@@ -0,0 +1,96 @@
+const { user } = require("pg/lib/defaults");
+const client = require("../client.js");
+
+const createReviews = async ({ creatorId, productId, description }) => {
+    try{
+        if (!description) {
+            throw{
+                name: "FieldRequired",
+                message: "Please provide the review."
+            }
+        }
+        const { rows: [reviews]} = await client.query(`
+            INSERT INTO reviews ("creatorId", "productId", description)
+            VALUES ($1, $2, $3)
+            RETURNING *;
+        `, [creatorId, productId, description]);
+        return reviews
+    }catch(error){
+    throw error;
+    }
+}
+
+const getReviews = async ({ description }) => {
+    try{
+        const { rows: [reviews] } = await client.query(`
+            SELECT *
+            FROM reviews
+            WHERE username=$1;
+        `, [description])
```

Comment:
why are we grabbing the review by the description when it is by the username?

Reply:
changed to getReviewsByUser by getting creatorId and removed getReviewsByUsername since both serve the same perps